frontend/promis.conf

```diff
@@ -31,6 +32,7 @@ server {
 
     # Django's static files (for backend's admin)
     location /static/ {
+        ${conf.web_root_config}
```

Comment:
Add more things like this if you need other paths conditionally handled too (i.e. switched between node.js server in dev and static files in production)